examples/ruby/README.md

```diff
@@ -0,0 +1,36 @@
+# Sara Alert FHIR Client App
+
+This simple example application written in the Ruby programming language illustrates how to authenticate and use the Sara Alert FHIR API, using the M2M workflow.
+
+First, follow the directions in the [wiki walk-through](https://github.com/SaraAlert/saraalert-fhir-ig/wiki/M2M-Walk-Through) to register your application.
```

Comment:
Updated this name so the link has changed: https://github.com/SaraAlert/saraalert-fhir-ig/wiki/Step-by-Step-Instructions-For-Local-Testing:-SMART-on-FHIR-Backend-Services-Workflow

Reply:
OK.

Reply:
Fixed.